### ISSUE TITLE
question: How to keep close bracket with last attribute?

### ISSUE THREAD

**Issue by kaangokdemir** [prettier/plugin-pug#82]
Hello I typed a pug line like this:

```pug
input(type='text' value='my_value' name='my_name' alt='my_alt' autocomplete='on')
```

Then run `prettier --write '**/*.pug'`

I've got this result after formatting complete:

```pug
input(
  type='text',
  value='my_value',
  name='my_name',
  alt='my_alt',
  autocomplete='on'
)
```

However I want to put close bracket at the end of my `autocomplete` attribute like this:

```pug
input(
  type='text',
  value='my_value',
  name='my_name',
  alt='my_alt',
  autocomplete='on')
```

Is there a way to do that or I need to change them manually? My `.prettierrc` file is like below, thank you.

```.json
{
  "semi": false,
  "singleQuote": true,
  "trailingComma": "all"
}
```

**Comment by Shinigami92** [comment]
Interesting style, but this would be a new feature request

**Comment by Shinigami92** [comment]
There is a bug 😕

If code is indented, spaces are preserved  
I will add a test, can you then add a new PR to fix it?

**Comment by Shinigami92** [comment]
https://github.com/prettier/plugin-pug/commit/ad4c6134a33504ddb973f6c664ceec6a9a79fa36

**Comment by Shinigami92** [comment]
Currently, more and more spaces / tabs are added before the closing bracket for each indentation

```diff
  div
    input(
      type="text",
      value="my_value",
      name="my_name",
      alt="my_alt",
-     autocomplete="on")
+     autocomplete="on"  )
```

**Comment by Shinigami92** [comment]
I have released [v1.3.0](https://www.npmjs.com/package/@prettier/plugin-pug/v/1.3.0)
Thanks for using my plugin
